uk.co.compucorp.civicrm.hrcore/gulp/tasks/requirejs.js

```diff
@@ -156,6 +157,27 @@ function requireJsMainTask (cb) {
   });
 }
 
+/**
+ * Changes the sourceMappingURL inside minified files from file path to URL path.
+ * This helps making source map files to be more easily located.
+ *
+ * @param {Function} cb
+ */
+function requireJsSourceMapPath (cb) {
+  var distFolder;
+  var extensionName = utils.getCurrentExtension();
+  var extensionPath = utils.getExtensionPath(extensionName);
```

Comment:
please check at least the documentation of the methods you use, `utils.getExtensionPath` already by default uses the current extension name if no name is passed as param